prelude.ftd

```diff
@@ -0,0 +1,2 @@
+-- import: fifthtry.github.io/Flash-theme as article
```

Comment:
Make this `as ft` and do a global replace of `article.` with `ft.` in all ftd files.